SPECS/libzip/libzip.spec

```diff
@@ -106,6 +113,10 @@ make check
 %{_libdir}/pkgconfig/libzip.pc
 
 %changelog
+* Mon Aug 19 2024 Andrew Phelps <anphel@microsoft.com> - 1.10.1-2
```

Comment:
The test build has a failure in a very weird way:

```bash
INFO[0072][scheduler] Passed SRPMs tests:                          
INFO[0072][scheduler] --> libzip-1.10.1-2.azl3.src.rpm             
INFO[0072][scheduler] Failed SRPMs:                                
INFO[0072][scheduler] --> libzip-1.10.1-2.azl3.src.rpm , error: exit status 2, for details see: /mnt/vss/_work/1/s/build/logs/pkggen/rpmbuilding/libzip-1.10.1-2.azl3.src.rpm.log 
```

Reply:
Yep, it looks like the build is passing when the check tests are enabled. But it fails when running without the tests with:
`"ERROR: No matching distribution found for nihtest"`
`nihtest` is not currently available in our specs, so it is installed using pip. But networking is only enabled when running tests, so pip cannot install during non-test builds.

Reply:
I've resolved this by adding a patch to modify CMakeLists.txt to skip the part where it disables regression tests when nihtest is not installed. This allows the tests to run, but also allows building without the tests.